config/config.go

```diff
@@ -203,13 +202,17 @@ func Get(cf string, ec *Extended, logger zap.Logger) (*AppConfig, error) {
 
 	cfg := AppConfig{
 		Logging:                b.Logging,
-		Verbose:                b.Verbose,
 		PIDPath:                b.Arachne.PIDPath,
 		Orchestrator:           b.Arachne.Orchestrator,
 		StandaloneTargetConfig: b.Arachne.StandaloneTargetConfig,
 		Metrics:                mc,
 	}
 
+	if cfg.Orchestrator.Enabled && cfg.StandaloneTargetConfig == "" {
```

Comment:
No, not a duplication because line 211 is about case for disabled orchestrator without specified target list file, whereas line 259 refers to the case where orchestrator mode has been enabled, but even after a target list file has been successfully fetched the latter is removed for whatever reason.